src/tests/color-spectrum.test.js

```diff
@@ -2,10 +2,23 @@ var should = require('should');
 var React = require('react/addons');
 var ColorSpectrum = require('../components/color-spectrum/color-spectrum.jsx');
 
+// https://github.com/facebook/react/issues/3721#issuecomment-106318499
+function shallowRender(makeComponent, context) {
```

Comment:
Ah, good call. Yeah i think this works for now. I'd love to spend some time improving tests specifically in a future sprint